src/govuk/components/notification-banner/notification-banner.test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env jest */
+
+const configPaths = require('../../../../config/paths.json')
+const PORT = configPaths.ports.test
+
+const baseUrl = 'http://localhost:' + PORT
+
+describe('/components/notification-banner/with-type-as-success', () => {
```

Comment:
These are great – well-written tests 👍🏻 

Do you think it'd be worth also having tests for the other two code paths?

- when the notification banner has a `data-disable-auto-focus` attribute, it's not focused on page load
- when the notification banner does not have role="alert", it's not focused on page load